Dockerfile

```diff
@@ -43,8 +43,11 @@ RUN apk add --no-cache \
   vo-amrwbenc-dev vo-amrwbenc-static \
   snappy snappy-dev snappy-static \
   xxd \
-  xz-dev xz-static
-
+  xz-dev xz-static \
+  asciidoc gdk-pixbuf-dev gflags-dev gtest-dev \
+  highway-dev lcms2-dev libjpeg-turbo-dev libpng-dev \
```

Comment:
Possible to use the exiting lcms2 build https://github.com/wader/static-ffmpeg/blob/master/Dockerfile#L273-L285? maybe have to change build order?

Reply:
highway-dev needed when using the bundled?

Reply:
libjpeg-turbo-dev libpng-dev installed above

Reply:
> Possible to use the exiting lcms2 build https://github.com/wader/static-ffmpeg/blob/master/Dockerfile#L273-L285? maybe have to change build order?

Just since didn't notice& forget to update those accordingly, will do the change.